### DISCUSSION TITLE
Settings app not working since 7/17 w/ GitHub Enterprise Cloud EMU

### DISCUSSION THREAD

**Discussion by enelsas** [repository-settings/app#985] [General]
We have been using the settings app to manage repo configurations for over a year. It appears to be no longer working since 7/17. Has anything changed on the GitHub side that would prevent it from working? If so, are there GitHub settings we need to change to make it work?

**Comment by travi** [comment]
https://github.com/repository-settings/app/releases/tag/v4.0.0 was released on the 16th. 

How are you using the settings app? Are you hosting your own instance? Are you using the hosted instance? Please answer the questions in the issue template

**Reply by travi** [reply]
if you are using the hosted instance, i believe https://github.com/repository-settings/app/commit/36df56224020440a4b47b817c2663c977bf43b98 fixes a problem with that instance. please confirm if that resolves your problem

**Reply by enelsas** [reply]
Yes, we are using hosted instance.  I just tested and issue appears to be resolved.  Thank you so much for the quick response.  Have a great weekend.

**Comment by enelsas** [comment]
## Problem Description

### What is actually happening

Repository-settings app does not appear to be working.

### What is the expected behavior

We maintain .github/settings.yml files in our repos.  When items in the settings.yml on main branch are updated repo configurations are updated in the repo hosting the file.

### Error output, if available

```
None.  I used to see activity for the settings app in audit log.  Not seeing any activity any more.
```

## Context

### Are you using the hosted instance of repository-settings/app or running your own?

We are using the the hosted instance of repository-settings.  It is installed on each organization.

### If running your own instance, are you using it with github.com or GitHub Enterprise?

#### Version of repository-settings/app

No way to tell.  I assume the installed app in our organization always pulls latest.

#### Version of GitHub Enterprise

We use GitHub Enterprise Cloud EMU (Enterprise-Managed-Users)
